### ISSUE TITLE
'Copy to' shortcut doesn't register, if the folder window is the focus

### ISSUE THREAD

**Issue by mcackay** [phaibin/MoliView#41]
Cmd-1 doesn't initiate a copy, if the window with the list of folders is the focus ... but if you then click back on the image itself, and repeat the shortcut, it works.

**Comment by phaibin** [comment]
Hi, I can't reproduce this error. MoliView will remember the last image focused when you open the "Copy/Move to" window. So it's not a problem when you focus the folder list window. 

Can you give me more details on how you encountered this problem? Thank you.

**Comment by mcackay** [comment]
To reproduce:

* Cmd-Shift-C to open Copy To window
* Cmd-2/etc and it should successfully copy to a target folder
* Click on the Copy To Window (which I had been doing sometimes to move it around slightly, or re-order the folder list)
* Cmd-2 ... now does nothing. No copy message, no 'beep' to indicate I've pressed an invalid shortcut/etc.

**Comment by phaibin** [comment]
https://github.com/user-attachments/assets/4bc7cea9-c1d5-4882-8e6b-76bdeabe709f

Hi, I made a screen record to show your steps. But I can copy without any problems. 😂

**Comment by mcackay** [comment]
I'm so confused -- I reproduced this multiple times yesterday, but can't today.
Let me try later and see if I can figure out the sequence. 

Perhaps close in the interim, and i'll reopen if can get you a video recording!

**Comment by mcackay** [comment]
ok, phew -- not going completely mad! Here's a video showing what I'm seeing.
It seems to be important that you don't click on the items in the Copy To window list first -- then the copy-to window focus seems to make a difference.

https://github.com/user-attachments/assets/e95af5e8-71c5-445c-b740-efd4176762ed

**Comment by phaibin** [comment]
I noticed that the first time the "Copy to" window was opened, the "Select..." button was focused. And if you click the folder list, the focus will move to the folder list. That's why the copy shortcut didn't work in the beginning. The shortcut will be sent to the "Select..." button.

Have you turned on some special settings in the system settings? Cause this situation will not happen on my computer and in my 15.1 virtual machine either.

The weirdest thing is, a normal button can't be focused, and even if I set it can be focused and it did get focused, the shortcuts can still work. So I guess this must be related to some of your settings.
